tests/Namshi/JOSE/Test/JWSTest.php

```diff
@@ -100,16 +97,6 @@ public function testVerificationRS256()
         $this->assertEquals('b', $payload['a']);
     }
 
-    public function testValidationOfAValidJWS()
```

Comment:
The tests are specifically hitting the `valid` method which is moved into the EasyJWS class, so the corresponding tests got moved into `EasyJWSTest`. The other tests in this file hit `verify` which is the important method for the base JWS class after these changes.